support/gathering-cluster-data.adoc

```diff
@@ -43,3 +43,14 @@ include::modules/support-collecting-network-trace.adoc[leveloffset=+1]
 
 // Providing diagnostic data to Red Hat Support
 include::modules/support-providing-diagnostic-data-to-red-hat.adoc[leveloffset=+1]
+
+// About `toolbox`
+include::modules/about-toolbox.adoc[leveloffset=+1]
+
+// Installing packages to a toolbox container
+[discrete]
```

Comment:
More of a question/opinion: why are the proceeding modules marked as `[discrete]`? IMHO, I found them to be informative and beneficial to be highlighted in the TOC for `toolbox`. I'll leave the choice up to you, but figured I'd flag it for consideration. :slightly_smiling_face:

Reply:
My instinct is that the `toolbox` command is overwhelmingly an intermediate step for troubleshooting.  I want the info available to customers, but not so visible as to make customer believe they need much knowledge of it.

Reply:
Makes sense to me; thanks for explaining